### ISSUE TITLE
My Treats

### ISSUE THREAD

**Issue by EhJay22** [OurTreatApp/OurTreat-iOS#154]
Is not showing the address of the company. When someone wins a prize they won't know which store to redeem it at. This is especially important for chains.

**Comment by anmolg** [comment]
That will be a change. The UI's never indicated a company address in my treats section. Please confirm, and this will be part of the 10 changes.

**Comment by EhJay22** [comment]
Yup, make the change.

**Comment by anmolg** [comment]
@shreyansjain - can you provide your time estimate for that?

**Comment by EhJay22** [comment]
Can wording be put in like "Prize can only be redeemed at (address)."?

**Comment by shreyansjain** [comment]
@anmolg can you return the address of the vendor in this URL: https://api.ourtreat.ca/customer/game_group/treat/48

**Comment by anmolg** [comment]
Should be resolved

**Comment by EhJay22** [comment]
Will this show up on a future build?

**Comment by shreyansjain** [comment]
Yes correct. It'll be in the next build

**Comment by EhJay22** [comment]
K

**Comment by shreyansjain** [comment]
@anmolg can you also add vendor id in the vendor object?

**Comment by EhJay22** [comment]
Close!
 Please have equal spacing between the congratulations line, the redemption address, and the Treat Finished. And make the address a hyperlink like when clicking on a Treat from the main page.

**Comment by shreyansjain** [comment]
![img_1726](https://user-images.githubusercontent.com/2359895/42488293-b1e9962e-83b9-11e8-9886-1fc1a774067e.PNG)
Looks good?

**Comment by shreyansjain** [comment]
Also, just to note, the clicking on the blue address will not do anything

**Comment by EhJay22** [comment]
Looks good!

**Comment by EhJay22** [comment]
The wording is there but the address is not.

![img_0015](https://user-images.githubusercontent.com/37785631/42518384-95105cce-8416-11e8-9d47-875ad163259d.PNG)

**Comment by shreyansjain** [comment]
Correct. There is a minor change needed in the backend, once done, it'll work then.

**Comment by EhJay22** [comment]
So, close this issue for now and create a new issue for the bug or leave this open?

**Comment by shreyansjain** [comment]
Leave this open, once @anmolg deploys his changes and you can test it then. If it works, close it then

**Comment by EhJay22** [comment]
Sounds good

**Comment by anmolg** [comment]
Should be deployed
